platforms/hyperledger-indy/configuration/roles/setup/identity/tasks/nested_main.yaml

```diff
@@ -0,0 +1,119 @@
+- name: Select Admin Identity for Organisation {{ component_name }}
+  shell: |
```

Comment:
Ensure this shell command works in sh shell, if not add executable as bash

Reply:
This works in ansible sh shell